dbt/adapters/singlestore/__version__.py

```diff
@@ -1 +1 @@
-version = "1.7.15"
+version = "1.7.1"
```

Comment:
I guess the patch version should the larger than 15. e.g. 16.

Reply:
it seems that using version 1.7.15 here was a mistake, given the comment in `setup.py `indicating that the versions should match between `setup.py` and `__version__.py`. using 1.7.1 would be more logical, especially since we currently have 1.7 listed on PyPI.

Reply:
sure this was a mistake. I'm just worried that __version__ file will have a smaller version. But pip shouldn't complain as it has 1.7 listed.